og_context/og_context.module

```diff
@@ -634,13 +634,9 @@ function _group_context_handler_entity($entity_type = 'node', $entity = NULL, $p
     return;
   }
 
-  list($id) = entity_extract_ids($entity_type, $entity);
```

Comment:
I don't think we can remove this. If we pass an unsaved entity, we need this validation.